Makefile

```diff
@@ -0,0 +1,8 @@
+CAIRO_0_PATH=cairo_programs/cairo_0
```

Comment:
do we have a strategy for cairo1 files as well?
with another cairo-compile binary? or rather scarb?
it can wait